src/components/new-request/search.vue

```diff
@@ -14,7 +14,7 @@
       <!-- New BC Business flow -->
       <template v-if="isNewBcBusiness">
         <EntityType v-if="getLocation" />
-        <CompanyType v-if="entity_type_cd" />
+        <CompanyType v-if="showCompanyTypeRadioButtons" />
```

Comment:
Do you want to take a stab at simplifying this getter?

```
get showCompanyTypeRadioButtons (): boolean {
  if (this.isNewBcBusiness && ...) return true
```

^^ The fallback return value should be False... probably need to comment out the existing code like I did for the other buttons.

Reply:
I have a better idea!

Instead of using the old getter above, include the logic right in the template. It should be simple for this flow. Try this:
```
<CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
```

Reply:
It works perfectly! So, are we going to get rid of that weird `showCompanyTypeRadioButtons`?

Also, this PR is incomplete. Apologies for that. I still need to handle the buttons.

Reply:
You said it -- it's a weird (gross) getter. I'd love to get rid of it.

Reply:
Sounds good.

Reply:
I "disabled" this getter in my previous PR (merged), so this conversation is obsolete.